### PR TITLE
[luci] ConvertNCHWToNHWC Pass for PAD Op

### PR DESCRIPTION
**Pull request by jinevening**
This supports PAD in ConvertNCHWToNHWCPass.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: #5401
Draft PR: #5438

### CODE REVIEW

File: compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp [hunk 1]
```diff
@@ -113,6 +113,43 @@ uint32_t cal_offset(const loco::TensorShape &dimension, const uint32_t *indices)
...
+        // get third dimension (H in NCHW)
+        data = paddings->at<loco::DataType::S32>(2 * 2 + i);
+      }
+      else if (dim == 2)
+      {
+        // get fourth dimension (W in NCHW)
+        data = paddings->at<loco::DataType::S32>(3 * 2 + i);
+      }
+
+      nhwc_paddings->at<loco::DataType::S32>(dim * 2 + i) = data;
```
Comment (mhs4670go) [inline compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp:147]: It seems that dim 0 and 3 will have data zero. Is it right?

Comment (jinevening) [reply compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp:147]: Yes, it is intended. Their values must be 0.

I'll add assert that checks the original data is also 0 for clarity.

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**